### DISCUSSION TITLE
Will RP Pico2 (RP2350) be supported?

### DISCUSSION THREAD

**Discussion by vasya-serega** [JAndrassy/ArduinoOTA#262] [Ideas]
Hi,
first attept to apply existing code was writted for RP Pico (RP2040) to RP Pico2 (RP2350) failed.
There is an error in VS Code + PlatformIO extention
![image](https://github.com/user-attachments/assets/65168b5d-3f60-463a-9953-e99fae9f603d)
It seems the newest RP2350 chip was not tested and is not supported yet.
So any plans to implement a support for Pico2 in the future?

**Comment by JAndrassy** [comment]
the Pico core has its own ArduinoOTA and now supports many networking chips. why do you want to use this library with it?

**Reply by vasya-serega** [reply]
> the Pico core has its own ArduinoOTA 
Can you please provide me with link?

> why do you want to use this library with it?
It is convinient. I can send firmwave over ethernet and then I have the data inside array let's call it _bytes use very simple code. And it works 
![image](https://github.com/user-attachments/assets/b4bb9ed9-1234-4820-8a7d-63d6587f8591)

**Reply by JAndrassy** [reply]
so there is PicoOTA library.
or adopt the RP2040 support and maybe even create a PR here.
